test/functional/test-performance.js

```diff
@@ -584,6 +591,8 @@ describes.realWin('performance', {amp: true}, env => {
           return whenFirstVisiblePromise.then(() => {
             expect(tickSpy.withArgs('pc')).to.be.calledOnce;
             expect(Number(tickSpy.withArgs('pc').args[0][1])).to.equal(0);
+            expect(getPerformanceMarks()).to.deep.equal(
```

Comment:
This unit test seems to be failing due to the order of `ofv` and `visible` not matching what is expected. Do we have a heuristic on which one of `ofv` and `visible` ticks first?  

```
        IT => should tick `pc` with `delta=0` when viewport is complete before user request document to be visible
          ✗ expected [ 'ol', 'pc', 'ofv', 'visible' ] to deeply equal [ 'ol', 'pc', 'visible', 'ofv' ]
```

Reply:
Its supposed to be deterministic in the test. If it isn't I'll fix the test.

Reply:
It is a bit creepy that this wasn't deterministic, but it showed that the `visible` mark wasn't even needed as `ofv` does have all the needed information, so this PR is a bit simpler now.